queries/ruby/highlights.scm

```diff
@@ -71,6 +84,10 @@
   "ensure"
 ] @keyword.exception
 
+(call
+  method: (identifier) @keyword
+  (#any-of? @keyword "abort" "at_exit" "exit" "fork" "loop" "trap"))
```

Comment:
These also need to be moved lower, since they are being overridden, and also probably `@function.builtin`